Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -288,9 +300,21 @@ extension Docc {
                     }
 
                 } else {
-                    throw TemplateOption.missingHTMLTemplateError(
-                        path: templateOption.defaultTemplateURL.path
+                    let missingTemplateDiagnostic = Diagnostic(
+                        severity: .warning,
+                        identifier: "org.swift.docc.MissingHTMLTemplate",
+                        summary: "Invalid or missing HTML template directory",
+                        explanation: """
+                            Invalid or missing HTML template directory, relative to the docc \
```

Comment:
Should we mention passing the `--no-transform-for-static-hosting` flag here to silence this warning?

Reply:
Good idea! Updated this.